packages/quantic/force-app/main/default/lwc/quanticNumberButton/quanticNumberButton.html

```diff
@@ -1,11 +1,8 @@
 <template>
-  <lightning-button
-    key={pageNumber}
-    variant={variant}
-    label={number}
-    title={number}
-    class="slds-m-left_xx-small"
-    value={number}
-    onclick={select}
-  ></lightning-button>
-</template>
+  <template if:true={selected}>
+    <button class="slds-button slds-button_brand slds-m-left_xx-small" aria-pressed="true" key={pageNumber} onclick={select}>{number}</button>
+  </template>
+  <template if:false={selected}>
+    <button class="slds-button slds-button_outline-brand slds-m-left_xx-small" aria-pressed="false" key={pageNumber} onclick={select}>{number}</button>
+  </template>
+</template>
```

Comment:
Instead of using conditional rendering, I'd suggest to add a property to return the CSS classes to apply. I think it would look simpler overall. It is currently hard to spot the differences between what is rendered when the button is pressed or not.

```html
<button class={buttonClasses} aria-pressed={selected} key={pageNumber} onclick={select}>{number}</button>
```

```javascript
get buttonClasses() {
  const classes = ['slds-button', 'slds-m-left__xx-small'];
  classes.push(this.selected ? 'slds-button_brand' : 'slds-button_outline-brand');
  return classes.join(' ');
}
```

Reply:
Removed conditional rendering and added getter for attribute class and aril-pressed.